catalyst_voices/lib/pages/registration/link_wallet/link_wallet_dialog.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:catalyst_cardano/catalyst_cardano.dart';
+import 'package:catalyst_voices/pages/registration/link_wallet/intro/link_wallet_intro_dialog.dart';
+import 'package:catalyst_voices/pages/registration/link_wallet/link_wallet_stage.dart';
+import 'package:catalyst_voices/pages/registration/link_wallet/select_wallet/select_wallet_dialog.dart';
+import 'package:catalyst_voices/widgets/modals/voices_dialog.dart';
+import 'package:flutter/material.dart';
+
+/// The link wallet flow consisting
+/// of [LinkWalletStage]'s during the registration.
+class LinkWalletDialog extends StatefulWidget {
+  const LinkWalletDialog({super.key});
```

Comment:
this constructor can be private so only `show` is access point here

Reply:
Fixed